jobs/pacman-awsrules/src/test/java/com/tmobile/cloud/awsrules/utils/IAMUtilsTest.java

```diff
@@ -60,4 +77,58 @@ public void getAccessKeyInformationForUserTest() throws Exception {
         
     }
     
+    @SuppressWarnings("static-access")
+    @Test
+    public void getAttachedPolicyOfIAMUserTest() throws Exception {
+        
+    	ListAttachedUserPoliciesResult policiesResult = new ListAttachedUserPoliciesResult();
+        
+        when(iamClient.listAttachedUserPolicies(anyObject())).thenReturn(policiesResult);
+        assertThat(iamUtils.getAttachedPolicyOfIAMUser("user",iamClient),is(notNullValue()));
+        
+    }
+    
+    @SuppressWarnings("static-access")
+    @Test
+    public void getActionListByPolicyTest() throws Exception {
+    	 AttachedPolicy attachedPolicies = new AttachedPolicy();
+         attachedPolicies.setPolicyName("IAMFullAccess");
+         List<AttachedPolicy> policies = new ArrayList<>();
+         policies.add(attachedPolicies);
+         
+         PolicyVersion versions = new PolicyVersion();
+         versions.setIsDefaultVersion(true);
+         versions.setVersionId("123");

```

Comment:
We can declare these values as constants and use these variables whenever needed  
 eg: declare it like ...private static final String  versionId = "123";
and use it as  ...
versions.setVersionId(versionId);